projects/aca-content/src/lib/components/knowledge-retrieval/search-ai/search-ai-results/search-ai-results.component.scss

```diff
@@ -1,3 +1,7 @@
+$light-grey1: #d5d5d5;
```

Comment:
can you move these 3 variables to variables.scss file for a case if anyone would like to reuse these colors in future?